src/main/resources/static/index.html

```diff
@@ -11,7 +11,7 @@
     <div class="topPanel">
         <h1>匿名回覆機器人</h1>
         [<b><a href="https://hackmd.io/@dfder/ByADyqspI" target="_blank">使用說明</a></b>]
-        [<b><a href="https://github.com/DF-wu/HideReplier" target="_blank">Github</a></b>]
+        [<b><a href="https://github.com/DF-wu/HideReplier" target="_blank">GitHub</a></b>]
         <b>←給個星星牡蠣一下好嗎？</b>
```

Comment:
- conflict
- plz merge the latest commit

Reply:
Solved by #10506f0